apps/test/unit/AzureTextToSpeechTest.js

```diff
@@ -4,7 +4,14 @@ import AzureTextToSpeech from '@cdo/apps/AzureTextToSpeech';
 
 const assertSoundResponsesEqual = (expected, actual) => {
   assert.deepEqual(expected.bytes, actual.bytes);
-  assert.deepEqual(expected.playbackOptions, actual.playbackOptions);
+  const playbackOptions = ['volume', 'loop', 'forceHTML5', 'allowHTML5Mobile'];
+  playbackOptions.forEach(opt => {
+    assert.deepEqual(
+      expected.playbackOptions[opt],
+      actual.playbackOptions[opt]
+    );
+  });
+  // assert.deepEqual(expected.playbackOptions, actual.playbackOptions);
```

Comment:
oops, didn't notice this before -- we should remove this assertion if we don't need it anymore

Reply:
oops, good catch